packages/fxa-auth-server/lib/senders/sms.js

```diff
@@ -65,18 +65,25 @@ module.exports = (log, translator, templates, config) => {
           },
           PhoneNumber: phoneNumber,
         };
+        const startTime = Date.now();
 
         return sns
           .publish(params)
           .promise()
           .then(result => {
+            if (statsd) {
+              statsd.timing('sms.send', Date.now() - startTime);
```

Comment:
Might be useful to split these. `sms.send.success` and `sms.send.error` as the log messages do.

Reply:
D'oh!  Just merged.  I'll created a follow-up PR.

Reply:
Addressed in #2767